bin/validate_lastnight_fba

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# requires source /project/projectdirs/desi/software/desi_environment.sh master
+from astropy.table import Table
+import numpy as np
+import os
+import argparse
+import sys
+
+parser = argparse.ArgumentParser()
+#required arguments
+parser.add_argument("--night", help="use this to specify the night, this is required unless you want 20210921 for some reason",default='20210921')
+#not required arguments
+parser.add_argument("--specprod",help="set specprod to something other than daily",default='daily')
+parser.add_argument("--outroot",help="the root directory for temporary outputs",default=os.environ['CSCRATCH'])
+args = parser.parse_args()
+
+month = args.night[:6]
+
+if args.night == '20210921':
+    print('DID YOU REALLY MEAN TO RUN THIS FOR '+args.night)
```

Comment:
Instead of using a hardcoded night that is probably not what you want, how about doing some datetime logic to derive the actual "lastnight" compared to the calendar day that you are running, e.g.
```python
if args.night is None:
    args.night = (datetime.datetime.now() - datetime.timedelta(days=1)).strftime('%Y%m%d')
```